server/lib/currency.js

```diff
@@ -18,6 +18,19 @@ function getDate(date = 'latest') {
   return date;
 }
 
+export function formatCurrency(currency, value) {
+  const _currency = currency.toUpperCase();
+  const currencyFormats = {
+    EUR: `€${value}`,
+    GBP: `£${value}`,
+    INR: `₹${value}`,
+    SEK: `kr ${value}`,
+    USD: `$${value}`,
+    UYU: `$U ${value}`,
+  };
```

Comment:
Generating all the possible strings to return only one seems overkill, it was good to have functions there and to only call the one that you're going to use.